app/js/daemonManager.js

```diff
@@ -73,7 +73,7 @@ function DaemonManager() {
 			if (err) {
 				self.Running = false;
 				// Check again later
-				setTimeout(updatePrompt, 1000);
+				setTimeout(updatePrompt, 100000); // Not great - we'll have to fix that.
```

Comment:
ohh I think I completely misunderstood what was happening here. 1 second is probably fine.